scripts/reporting/ncanda_quality_control_script.py

```diff
@@ -300,12 +301,11 @@ def main(args):
         for e in error:
     		if e == 'null':
     			error.remove(e)
-        f = csv.writer(open("missing_form.csv", "wb+"))
-    	f.writerow(["subject_site_id", "visit_date", "np_missing", "event_name", "error"])
+        f = csv.writer(open(args.csvdir, "wb+"))
+    	f.writerow(["subject_site_id", "visit_date", "event_name", "error"])
     	for x in error:
       	 	f.writerow([x["subject_site_id"],
         	   	        x["visit_date"],
```

Comment:
similarly, all the `x[foo]` statements should all be at the same level of indentation.